spec/spec_helper.rb

```diff
@@ -43,3 +40,19 @@ def log
     @log.string
   end
 end
+
+class GelfMock < Logger
```

Comment:
`spec_helper` doesn't seem the right place to define this - can we move it to a dedicated file somewhere?

Reply:
Applied